util/todo.ts

```diff
@@ -13,6 +13,40 @@ export async function createTodo(request:APIRequestContext,body:{title:string,st
 }
 
 export async function deleteTodo(request:APIRequestContext,id:number){
-    const respons = await request.delete(`/v1/todo/${id}`)
-    return await respons.status()
+    const resp = await request.delete(`/v1/todo/${id}`)
+    return resp.status()
+}
+
+export async function getTodo(request:APIRequestContext,id?:number) {
+    
+ 
+        const res = await request.get(`/v1/todo/${id}`)
+        return {status:res.status(),body: await res.json()}
+    
+}
+    
+export async function getTodoAll(request:APIRequestContext) {
+    
+        const resp = await request.get("/v1/todo")
+     
+        return {status:resp.status(),body: await resp.json()}
+}
+export async function patchTodo(request:APIRequestContext,body:{title?:string,status?:string}){
+    const respons = await request.patch("/v1/todo",{
+        data: body,
+        headers:{
+         'content-Type': 'application/json'
+        }
+    })
+    return {status:respons.status(),body: await respons.json()} 
+}
+
+export async function putTodo(request:APIRequestContext,body:{title?:string,status?:string}){
+    const respons = await request.put("/v1/todo",{
+        data: body,
+        headers:{
+         'content-Type': 'application/json'
+        }
+    })
+    return {status:respons.status(),body: await respons.json()}  
```

Comment:
Where this two functions are used? if your peer is working on this, she should add those not you.

Reply:
yes.. she would be working. I just created unknowingly